### DISCUSSION TITLE
Awesome list of TQEC resources

### DISCUSSION THREAD

**Discussion by inmzhang** [tqec/tqec#247] [General]
I came across the idea that we can create an awesome list of TQEC resources.

As a community, our primary goal is to learn more about TQEC, which will then enable us to develop tools based on our knowledge. Additionally, I believe many community members are researchers or students who might want to delve into the theory rather than focusing on specific tasks like coding. Creating a resource list would benefit everyone by providing a shared repository for learning, querying, and citing.

The resource list could take the form of a separate GitHub repository or a folder in our shared Google Drive. What do you think?

**Comment by inmzhang** [comment]
Tag here to ensure notifying all @QCHackers/tqec.

**Comment by nelimee** [comment]
I remember a few interesting links, I think an awesome-like repository already exists for QEC, but I cannot find it back, so it might have been removed. Also, there is the fantastic https://errorcorrectionzoo.org/.

**Comment by inmzhang** [comment]
This is an initial sketch of what I wanted to have: https://github.com/inmzhang/awesome-tqec.

I'm not sure whether you think this will be helpful. I understand the main focus in our group is to build the automation tool, but a 
more broad view of the TQEC field can also be helpful.

**Comment by afowler** [comment]
Thanks Yiming, I think this is great. I do think people will find it
helpful.
